src/components/Categories.js

```diff
@@ -1,7 +1,24 @@
-const Category = () => (
-  <div>
-    <button type="button">check status</button>
-  </div>
-);
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkBookStatus } from '../redux/categories/categories';
+
+const Category = () => {
+  const dispatch = useDispatch();
+
+  const checkStatus = () => {
+    dispatch(checkBookStatus());
+  };
+
+  return (
+    <div>
+      <button
+        type="button"
+        onClick={(e) => checkStatus(e)}
+      >
+        check status
+      </button>
+    </div>
+  );
+};
```

Comment:
- [ ] When I click on the `check status` button the right action is dispatched correctly but please make the `Under construction` phrase appear on the page that's what suppose to happen.

Reply:
done as requested